### PR TITLE
Revert changes to systemctl (b35ec85)

### PR DESCRIPTION
**Pull request by rwx788**
As a part of refactoring, we have replaced systemctl calls with wrapper
function. Whereas for multi_users_dm test suite, we won't get return
code.

Doesn't fix the test though, as it break due to https://bugzilla.opensuse.org/show_bug.cgi?id=1063060 

Revert [PR#3263](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3263).

### CODE REVIEW

**Comment by rwx788** [comment]
@okurz, please, take a look.

File: tests/x11/multi_users_dm.pm [hunk 1]
```diff
@@ -20,15 +20,15 @@ use testapi;
 use utils;
 
 sub ensure_multi_user_target {
-    systemctl 'isolate multi-user.target';
+    type_string "systemctl isolate multi-user.target\n";
```
Comment (okurz) [inline tests/x11/multi_users_dm.pm]: I don't understand, why is this necessary?

Comment (rwx788) [reply tests/x11/multi_users_dm.pm]: That's the reason why I've put the label. But this changed the behavior, as with assert_script_run command fails, even though command worked and we see login screen. So I will take a closer look why we don't get anything on the serial.

**Comment by coolo** [comment]
this looks done to me

**Comment by okurz** [comment]
"done" as in … not needed?

**Comment by coolo** [comment]
"done" as in discussion completed

**Comment by rwx788** [comment]
@okurz the point here is that there is no exit code. Test fails due to known bug, but with your recent change it fails one step before and I'm not sure what is expected when changing runlevel. But we can clearly see that command works fine and login shell is there.

File: tests/x11/multi_users_dm.pm (hunk 1 above)
Comment (okurz) [inline tests/x11/multi_users_dm.pm]: I would have preferred a description here to describe why the command `systemctl` does not work. Otherwise it will not be clear to a person in the future why he should not change this.
